README.md

```diff
@@ -364,6 +364,31 @@ func ExampleDatabase_QueryHistoricState() {
 ```
 </details>
 
+# Performance 
+
+Carmen is the backbone of [Fantom Sonic](https://fantom.foundation/sonicPage) blockchain. 
+It is the key component that enables high performance in a disk space conservative manner. 
+
+Carmen was extensively tested as part of the Sonic client (i.e. a blockchain node). 
+Three main scenarios were tested:
+1. **Realistic traffic:** a mix of transactions that resemble realistic traffic, such as token transfers, token mints, and complex multi-step swaps.
+2. **Token swaps:** transactions calling the uniswap’s contract, which handles multi-step swaps between tokens, demonstrating what the next generation of DEXs can achieve.
+3. **Token transfers:** transactions processing only ERC-20 transfers, demonstrating usage for next-generation wallets, payment providers.
+
+The client could process thousands of transactions per second as depicted in the table 
+and further detailed in [a blog post](https://blog.fantom.foundation/3-incredible-performances-from-fantom-sonic-closed-testnet/).
+
+| Configuration | Speed Tx/s |
+| ------------- | ---------- |
+| Realistic       |  2000 |
+| Token swaps     |  4000 |
+| Token transfers | 10000 |
```

Comment:
Do we have numbers for the `geth` DB?

Reply:
no I dont, I will ask Peter. Would be good to compare, but not sure if anyone run this experiment.

Reply:
I believe thats the Norma transaction generator, is that correct? If so I will do a measurement and send it to you @kjezek

Reply:
@petr-hanzl @HerbertJordan shall we close this comment and merge the PR? If we cannot add  the numbers quickly  I think it is better to add them later instead of keeping this PR around for weeks.

Reply:
Lukas has fixed the tx generator. I will provide the numbers within today